cinema/tests/test_movie_api.py

```diff
@@ -42,21 +40,15 @@ def test_get_movies(self):
             self.assertEqual(movies.data[0][field], titanic[field])
```

Comment:
This assertion checks that the movie details are correctly retrieved, ensuring the GET operation works as expected.